infoFileComparison/src/main/scala/za/co/absa/hermes/infoFileComparison/config/ManualConfig.scala

```diff
@@ -0,0 +1,6 @@
+package za.co.absa.hermes.infoFileComparison.config
+
+class ManualConfig(
```

Comment:
Is this used? An instance of `TypeSafe` config can be created from a `Map` (will add a comment below).

Reply:
ManualConfig was created for the instances when Hermes is used as a library